<h1>pandas.errors.DataError</h1> <dl class="py exception"> <dt class="sig sig-object py" id="pandas.errors.DataError"> <em class="property"><span class="pre">exception</span></em><span class="sig-prename descclassname"><span class="pre">pandas.errors.</span></span><span class="sig-name descname"><span class="pre">DataError</span></span><a class="reference external" href="https://github.com/pandas-dev/pandas/blob/v1.5.0/pandas/errors/__init__.py#L240-L246"><span class="viewcode-link"><span class="pre">[source]</span></span></a>
</dt> <dd>
<p>Exceptionn raised when performing an operation on non-numerical data.</p> <p>For example, calling <code class="docutils literal notranslate"><span class="pre">ohlc</span></code> on a non-numerical column or a function on a rolling window.</p> </dd>
</dl><div class="_attribution">
  <p class="_attribution-p">
    &copy; 2008&ndash;2022, AQR Capital Management, LLC, Lambda Foundry, Inc. and PyData Development Team<br>Licensed under the 3-clause BSD License.<br>
    <a href="https://pandas.pydata.org/pandas-docs/version/1.5.0/reference/api/pandas.errors.DataError.html" class="_attribution-link">https://pandas.pydata.org/pandas-docs/version/1.5.0/reference/api/pandas.errors.DataError.html</a>
  </p>
</div>
